org.mfat.systemdpilot/org.mfat.systemdpilot.yml

```diff
@@ -0,0 +1,37 @@
+app-id: org.mfat.systemdpilot
+runtime: org.gnome.Platform
+runtime-version: '45'
+sdk: org.gnome.Sdk
+command: systemd-pilot
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --system-talk-name=org.freedesktop.systemd1
+  - --system-talk-name=org.freedesktop.login1
+  - --socket=session-bus
+  - --system-talk-name=org.freedesktop.PolicyKit1
+  - --socket=system-bus
```

Comment:
session-bus and system-bus are not allowed. Figure out what bus names you need https://docs.flatpak.org/en/latest/debugging.html#audit-session-or-system-bus-traffic